src/elements/content-sidebar/activity-feed/__tests__/Avatar-test.js

```diff
@@ -9,66 +9,57 @@ describe('elements/content-sidebar/ActivityFeed/Avatar', () => {
         name: 'foo bar',
         type: 'user',
     };
-    const getAvatarUrl = jest.fn().mockReturnValue(Promise.resolve('foo'));
 
     const getWrapper = props => shallow(<Avatar {...props} />);
 
-    test('should render nothing if no avatarUrl in state and getAvatarUrl method was passed', () => {
-        expect(
-            getWrapper({ user, getAvatarUrl })
-                .dive()
-                .find('Avatar')
-                .exists(),
-        ).toBe(false);
-        expect(getAvatarUrl).toBeCalledWith(user.id);
-    });
-
-    test('should render avatar with initials if getAvatarUrl is not passed in and no avatarUrl is in state', () => {
-        expect(
-            getWrapper({ user })
-                .find('AvatarInitials')
-                .exists(),
-        ).toBe(false);
-        expect(getAvatarUrl).not.toBeCalledWith(user.id);
+    test('should pass isPending prop to wrapped Avatar getAvatarUrl prop is passed', () => {
+        const getAvatarUrl = jest.fn().mockResolvedValue('avatar.jpg');
+        const wrapper = getWrapper({ user, getAvatarUrl });
+        const wrappedAvatar = wrapper.find('Avatar').first();
+        expect(wrappedAvatar.props()).toEqual(expect.objectContaining({ isPending: true }));
+        expect(wrapper).toMatchInlineSnapshot(`
```

Comment:
can we add these to the snapshots file since this component already had one?

Reply:
The file is empty/deleted now, the inline snaps replaced them

Reply:
What was the reason of making it inline? It will be harder to diff if snapshots are inlined and large.

Reply:
Inline snaps are more likely to be read and understood. You shouldn't use them if the diff is large, but you shouldn't be using snapshots at all if they are _huge_.

Reply:
And why not? Even if the snapshot is 2x-3x of this one, it provides usefulness. Please move this out to its own file as thats what we have for other files and other projects.

Reply:
The Jest ESLint plugin has some good reasoning for these rules:

[no-large-snapshots](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-large-snapshots.md)

> When using Jest's snapshot capability one should be mindful of the size of created snapshots. As a best practice snapshots should be limited in size in order to be more manageable and reviewable. A stored snapshot is only as good as its review and as such keeping it short, sweet, and readable is important to allow for thorough reviews.

[prefer-inline-snapshots](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-inline-snapshots.md)

> In order to make snapshot tests more managable and reviewable toMatchInlineSnapshot() and toThrowErrorMatchingInlineSnapshot should be used to write the snapshots inline in the test file.